app/js/scripts.js

```diff
@@ -44,6 +44,20 @@ window.onload = function() {
 		}
 	}
 
+	
+	addInfo.addEventListener('click', showTextarea);
+
 }
 
 
+
+function showTextarea(e) {

```

Comment:
this function should be named toggleTextArea because it can both show and hide element.